success-stories/index.html

```diff
@@ -1,8 +1,8 @@
 ---
 layout: success-stories 
 title:  "Success Stories"
-intro: "Real world use cases from global software enterprise companies and start ups about the impact of using Spinnaker in their organizations."
-description: "Click on the box to learn how developers are the key change agents in their companies."
+intro: "Real world use cases about the transformative power of Spinnaker."
```

Comment:
One nit: this isn't actually a grammatical sentence, so I vote to leave the period off. (Where's the blushing smiley when I need it GitHub?)